CMakeLists.txt

```diff
@@ -111,6 +111,13 @@ if(TC_BUILD_IOS)
   )
   SET(CMAKE_OSX_SYSROOT "${IOS_ISYSROOT}")
   SET(COMPILER_FLAGS "${COMPILER_FLAGS} -Wno-c++11-narrowing -isysroot ${IOS_ISYSROOT}")
+elseif(APPLE)
+  EXECUTE_PROCESS(
```

Comment:
We should probably refactor this whole block to share logic with the `xcrun` command around line 735.